Dockerfile

```diff
@@ -1,17 +1,25 @@
-FROM oconnormi/ddf-base:edge
+FROM oconnormi/ddf-base
 MAINTAINER oconnormi
 
 ENV APP_VERSION=0.2
 ENV APP_BASE=/opt
 ENV APP_HOME=$APP_BASE/alliance
 ENV APP_LOG=$APP_HOME/data/log/ddf.log
+#releases, snapshots
+ENV APP_REPO=releases
+ENV STARTUP_APPS=admin-app;platform-app;catalog-app;spatial-app;security-services-app;security-app
+ENV UNINSTALL_FEATURES=admin-modules-installer
```

Comment:
Do the other admin modules get installed, i.e. `configuration`, `sources`...?
I can't remember if the apps do it or if you need to have the `admin-post-install-modules` install instead